### PR TITLE
integrate gorelease

### PR DESCRIPTION
**Pull request by asabya**

### CODE REVIEW

File: .goreleaser.yaml [hunk 1]
```diff
@@ -126,4 +126,71 @@ archives:
...
+    dockerfile: Dockerfile.goreleaser
+    build_flag_templates:
+      - "--platform=linux/amd64"
+      - "--pull"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.title={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+  - image_templates:
+      - "asabya/dfs:v{{ .Version }}-armv7"
```
Comment (darkobas2) [inline .goreleaser.yaml:146]: Asabya or fds ?

File: Dockerfile.goreleaser [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.15
+
+ARG CONFIG
+ENV CONFIG=$CONFIG
+
```
Comment (darkobas2) [inline Dockerfile.goreleaser:5]: We dont need complicate with config if we provide seperate dockerfile

File: Dockerfile.goreleaser [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+RUN addgroup -g 10000 fds
+RUN adduser -u 10000 -G fds -h /home/fds -D fds
+
+USER fds
+RUN if [ -n "$CONFIG" ]; then  echo -e "$CONFIG" > ~/.dfs.yaml; fi
```
Comment (darkobas2) [inline Dockerfile.goreleaser:10]: Not needed

**Comment by darkobas2** [comment]
Can we fix the go lint errors?

File: Dockerfile.goreleaser (hunk 2 above)
Comment (asabya) [reply Dockerfile.goreleaser:5]: This doesn't include the config file in the docker. Should we include it?

**Comment by tfius** [comment]
merge when ready

**Pull request merged**